webpack/base.config.js

```diff
@@ -20,8 +23,8 @@ const baseConfig = {
   externals: undefined,
 
   module: {
-    loaders: [
-      { test: /\.js/, loader: jsLoader, exclude: /node_modules/ }
+    rules: [
+      { test: /\.js/, use: [jsLoader], exclude: /node_modules/ }
```

Comment:
Oh, I assumed that an array was required there. I guess I saw it being used like that in an example. Want me to change it?